src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -69,7 +71,8 @@ public async Task<IActionResult> OnGetCallbackAsync(string returnUrl = null, str
                 return RedirectToPage("./Login", new { ReturnUrl = returnUrl });
             }
 
-            var info = await _signInManager.GetExternalLoginInfoAsync();
+            Task<ExternalLoginInfo?> idk = _signInManager.GetExternalLoginInfoAsync();
```

Comment:
OH GOD, please revert back to the old code, this was just for testing with the debugger :'D

Reply:
I have changed it back to what it was before